redaxo/src/core/lib/util/editor.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+class rex_editor
+{
+    use rex_factory_trait;
+
+    // see https://github.com/filp/whoops/blob/master/docs/Open%20Files%20In%20An%20Editor.md
+    // keep this list in sync with $supportedEditors in system.settings.php
+    private $editors = [
+        'atom' => 'atom://core/open/file?filename=%file&line=%line',
+        'emacs' => 'emacs://open?url=file://%file&line=%line',
+        'idea' => 'idea://open?file=%file&line=%line',
+        'macvim' => 'mvim://open/?url=file://%file&line=%line',
+        'phpstorm' => 'phpstorm://open?file=%file&line=%line',
+        'sublime' => 'subl://open?url=file://%file&line=%line',
+        'textmate' => 'txmt://open?url=file://%file&line=%line',
+        'vscode' => 'vscode://file/%file:%line',
+    ];
+
+    public function urlFromFile($filePath, $line)
```

Comment:
Ist es nicht eher `urlForFile`?
Oder einfach nur `url()`? Oder vielleicht `getUrl($file, $line)`, so heißen ja in der Regel unsere url-Methoden.

Reply:
Done